### PR TITLE
Login screen

### PR DESCRIPTION
**Pull request by Vojtech-Sassmann**
* Implemented login screen for the AdminGui. This login screen is shown
when user is not logged in and the config property `auto_auth_redirect`
is set to false.
* Login screen also added to the user profile, HOWEVER! it is not
finished. The app component has to be adjusted so the sidemenu is
hidden.

### CODE REVIEW

**Review by xkureck — CHANGES_REQUESTED** [review]
image
Some sort of text would be nice above the login button. Also consider using the "sign in" label on the button
image
FIx lint

File: apps/admin-gui/src/app/app.component.html [hunk 1]
```diff
@@ -2,39 +2,54 @@
...
-               [ngStyle]="{'border-color': sideBarBorderColor}"
-               [mode]="sidebarMode"
-               [opened]="!isMobile()"
-               [fixedInViewport]="true"
-               [fixedTopGap]="getTopGap()">
-    <perfect-scrollbar [ngStyle]="{'background': sideMenubackgroundColor}" [scrollIndicators]="true">
-      <app-side-menu [sideNav]="sidenav"></app-side-menu>
-    </perfect-scrollbar>
-  </mat-sidenav>
+<ng-template [ngIf]="!isLoginScreenShow">
```
[the remark this answers is not in the document]
Comment (Vojtech-Sassmann) [reply apps/admin-gui/src/app/app.component.html:6]: Hi @xkureck, I have refactored the login screen. Now it should be easier to add it to other apps.

File: libs/perun/login/src/lib/login-screen/login-screen.component.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class="container login-con vo-theme">
+  <div class="row">
+    <div class="ml-auto mr-auto">
+      <button mat-raised-button
+              color="primary"
+              (click)="startAuth()">
+        Login
```
Comment (xkureck) [inline libs/perun/login/src/lib/login-screen/login-screen.component.html:7]: localization

**Review by xkureck — COMMENTED** [review]
image
Did you considered this or do you think we just dont need it?

**Comment by Vojtech-Sassmann** [comment]
> image
> Did you considered this or do you think we just dont need it?

Sorry, I didn't notice the comment. I will add the text and also change the label.

**Review by xkureck — APPROVED** [review]

**Pull request merged**